### PR TITLE
Dum 693 update dabp documentation

### PR DESCRIPTION
**Pull request by theishinz-onegini**

### CODE REVIEW

**Review by aandrzejewski — CHANGES_REQUESTED** [review]

File: docs/dabp/guides/concepts/permissions-in-depth.md [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
-In total are there 6 permissions:
+In total, there are 6 permissions:
 
 - **Invite/remove group members** enables the user to add, edit, and delete members of the group, and its subgroups.
 - **Assign permissions to group members** enables the user to add, edit, and delete permissions of group members.
 - **Assign policies to group members** enables the user to add, edit, and delete policies of group members.
 - **Add/edit/delete groups** enables the user to add, edit, and delete subgroups
 - **Assign policies to groups** enables the user to add, edit, and delete policies on a group level.
 - **Add/edit/delete policies** enables the user to add, edit, and delete policies to the system.
+This permission can also get on members of the root group.
```
[the remark this answers is not in the document]
Comment (theishinz-onegini) [reply docs/dabp/guides/concepts/permissions-in-depth.md:13]: No I mean that you only can assign the permission on members of the root group. So: https://onegini.atlassian.net/browse/DUM-678

[the remark this answers is not in the document]
Comment (aandrzejewski) [reply docs/dabp/guides/concepts/permissions-in-depth.md:13]: Ok, so I would rewrite this to something like this `This permission can be assigned only to members of the root group.` ?

File: docs/dabp/guides/functional/manage-policy-via-ui.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+That is the top-level group. 
+The "Policies" page is available under "Settings" when you have this permission.
+
+![Policies page](../../img/policies-page.png)
+
+## Create policies
+*For technical documentation, see here [how to create a new policy via the API](../technical/create-policy-via-api.md).*
+
+You can create a new policy using the "New Policy" button on the page. You have to give the policy a unique name.
+A new policy is automatically added to the root group. Therefore a member of the root group can assign it to other groups.
```
[the remark this answers is not in the document]
Comment (rvroon) [reply docs/dabp/guides/functional/manage-policy-via-ui.md:16]: `A new policy is automatically added to the root group. Thus an administrator can assign it to other groups.`

File: docs/dabp/guides/functional/manage-policy-via-ui.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Manage policies
+
+You need to create a policy before you can assign it to groups and members.
+Policies can be created on the "Policies" page or via the [Create Policy API](../technical/create-policy-via-api.md). 
+
+To manage policies, you must be an administrator with the `Add/edit/delete policies` permission on the root group. 
+That is the top-level group.
```
Comment (rvroon) [inline docs/dabp/guides/functional/manage-policy-via-ui.md:7]: We defined an administrator as a "user with all permissions on the root group", therefore you cannot be an `administrator on the root group`. So we should use `superuser` here.

File: docs/dabp/guides/concepts/permissions-in-depth.md [hunk 4]
```diff
@@ -1,21 +1,24 @@
...
 
-If a person has any permissions for any (sub)group we refer to that user as a superuser.
-If a person has all permissions on the root level we refer to that user as an administrator.
+If a person has any permissions for any (sub-)group, we refer to that user as a superuser.
```
[the remark this answers is not in the document]
Comment (theishinz-onegini) [reply docs/dabp/guides/concepts/permissions-in-depth.md:15]: Your right. Changed it back.

File: docs/dabp/guides/functional/manage-policy-via-ui.md (hunk 3 above)
Comment (theishinz-onegini) [reply docs/dabp/guides/functional/manage-policy-via-ui.md:7]: I will change it to  'user' in order to keep it general.

File: docs/dabp/guides/functional/manage-policy-via-ui.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (theishinz-onegini) [reply docs/dabp/guides/functional/manage-policy-via-ui.md:16]: Changed

**Review by aandrzejewski — APPROVED** [review]

**Pull request merged**